src/Layouts/Partials/Layer.vala

```diff
@@ -518,7 +518,7 @@ public class Akira.Layouts.Partials.Layer : Gtk.ListBoxRow {
     }
 
     public bool update_on_escape (Gdk.EventKey key) {
-        if (key.keyval == 65307) {
+        if (key.keyval == Gdk.Key.Search) {
```

Comment:
This should be the `Esc` key.
If you're changing this, please update also the same method in the `Artboard.vala` file